packages/bits/spec/components/paginator/paginator.e2e.ts

```diff
@@ -44,6 +42,8 @@ describe("USERCONTROL paginator", () => {
         expect(await atom.isActivePage(1)).toBe(false);
         expect(await atom.isActivePage(3)).toBe(false);
         expect(await atom.isActivePage(4)).toBe(true);
+        await atom.pageLinkClick(1);
```

Comment:
I assume this is to restore the paginator to its default state? If so, I like your comments to that effect in your other PRs. Do you mind adding those in this suite too?

Reply:
Sure, adding.